test/api/v01/request_helper.rb

```diff
@@ -118,6 +119,7 @@ def asynchronously(options = {start_worker: false})
     OptimizerWrapper.config[:solve][:synchronously] = false
     old_resque_inline = Resque.inline
     Resque.inline = false
+    sleep 0.5
```

Comment:
why do we need an extra sleep here?

Reply:
Maybe this is not the best solution. Some tests fail because of "job_id not found". I run some (small) tests and when adding a sleep here we reduce number of times this occurs : 
Without sleep            test fails 8/11 times because of this, 
With sleep                 test fails 3/11 times because of this.

I assumed that we try to get result too fast because it has not arrived to worker yet. But now that I look a little closer to this, I guess sleep should rather be in wait status.

Reply:
In fact we do have one in `wait_status_csv` but not in `wait_status`

Reply:
I specifically looked into the sleep problem not long ago. The sleep shouldn't be needed. 

If you are doing this because of those tests with `job_id=xx not found` problem, they are related to a "zombie" worker at the background, if you check the xx job_id that can't be found, that job is already launched by another "zombie" worker and then "cleaned" by the worker who supposed to do the job.

If this is because of another test, lets find the real culprit, sleep is just hiding the issue.

Reply:
Thank you, I wasn't really sure that was needed.